lib/registrant.rb

```diff
@@ -0,0 +1,27 @@
+class Registrant
+  attr_reader :name, 
+              :age, 
+              :permit, 
+              :license_data
+
+  attr_accessor :license_data
+
+  def initialize(name, age, permit = false)
+    @name          = name
+    @age          = age
+    @permit       = permit
+    @license_data = {
+                    written: false,
+                    license: false,
+                    renewed: false
+                    }
+  end
+
+  def permit?
+    @permit
+  end
+
+  def earn_permit
+    @permit = true
```

Comment:
You could make this more robust by adding a guard statement
```ruby
@permit = true if @age >= 15
```

Reply:
What is a guard statement?
I guess i'm unfamiliar with whats "allowed" to be an instance variables value.

Reply:
The guard statement is the `if` part of what I showed you. Which is basically saying set `@permit` to true if they a registrant is of the specified age.  I am not sure what you mean by `I guess I'm unfamiliar with whats "allowed" to be an instance variable value.`
You were correct in setting `@permit = true`. I was just showing you something.